x-pack/plugins/security_solution/public/cases/components/all_cases/index.test.tsx

```diff
@@ -97,23 +96,16 @@ describe('AllCases', () => {
   });
   /* eslint-enable no-console */
   beforeEach(() => {
-    jest.resetAllMocks();
+    jest.clearAllMocks();
```

Comment:
I left this one instance of `clearAllMocks` as detailed in 0e5dfe37bcd059600ad2d827d1f5734db62fc06d. Instead of defining mocks once and then clearing their state in a `beforeEach`, my preference is to instead define the mocks in the `beforeEach`. That's the pattern I followed for the other tests, this one was the exception.